README.md

```diff
@@ -1,2 +1,61 @@
-# corporate-theme
-StoreConnect Corporate Theme
+# Corporate Theme
+
+This is a multi-purpose theme, ideal for corporate, B2B, or services companies.
+
+## Directory Structure
+
+This theme has a specific folder structure. Some folders may be empty, but they need to remain part of the theme structure as they are not overridden or used for new templates.
+
+Also, it has been developed using the [Base Theme](https://github.com/GetStoreConnect/base-theme) as a boilerplate. The Corporate theme doesn't contain all the base theme structure, instead, we only include those customised, modified and new templates, snippets, variables or translations. Feel free to create a brand new theme using the base theme.
+
+## Theme Installer
+
+The theme installer is in your Salesforce org, where you must upload this theme as a zip file. Before uploading, remove the .git directory and README.md file from this repo. The theme installer displays errors if it does not find the standard directory structure.
+
+## New Templates
+
+You can create new templates as long as they reside within the standard directory structure.
+
+## Features
+
+- Full width and height hero banners
+- Minimalistic header
+- Animated image with content overlay
+- Featured products and categories slider
+- Featured articles cards
+- Improved container layout 100% responsive
+- Page top banners
+- Corporate footer style
+
+### New Content Block
+
+- hero
+
+### Modified Content Blocks
+
+- container
+- featured_articles
+- featured_categories
+- featured_products
+
+## How to Add Content Blocks to Picklist
+
+You can follow the instructions in this document to [add content blocks to the picklist](https://help.getstoreconnect.com/documentation/adding-templates-to-content-template-picklist.html).
+
+## Configuration
+
+- Ensure that your root directory does not contain any `.git` hidden folders or files
```

Comment:
check this but don't think this step is necessary. Seems only requirement is that root folder is named `theme`

Reply:
@toddweetman we should keep it as it if for the same reason mentioned above.